packages/integrations/mdx/test/mdx-syntax-highlighting.test.js

```diff
@@ -64,4 +67,25 @@ describe('MDX syntax highlighting', () => {
 			expect(prismCodeBlock).to.not.be.null;
 		});
 	});
+
+	describe('disabled', () => {
```

Comment:
I think naming this section "disabled" is misleading as the tests are still being run and what you're checking is if custom highlighters work. I'd suggest calling this "custom highlighters" instead.

Reply:
Fair enough! I'll just remove that `describe` wrapper. Doesn't add anythign